gpkit/constraints/costed.py

```diff
@@ -24,10 +24,12 @@ def subinplace(self, subs, value=None):
         self.cost = self.cost.sub(subs, value)
         ConstraintSet.subinplace(self, subs, value)
 
-    @property
-    def varkeys(self):
+    def reset_varkeys(self, init_dict=None):
         "return all Varkeys present in this ConstraintSet"
```

Comment:
the method changed but the docstring did not -- is the docstring still accurate?